packages/slate-react/src/components/content.js

```diff
@@ -151,7 +151,7 @@ class Content extends React.Component {
 
     // COMPAT: In Firefox, there's a but where `getSelection` can return `null`.
     // https://bugzilla.mozilla.org/show_bug.cgi?id=827585 (2018/11/07)
-    if (!native) {
+    if (!native || editor.isComposing()) {
```

Comment:
I think instead of not reflecting the update here, we might want to instead not handle `onSelect` in the first place? Such that the there are no `set_selection` operations that are queued from composition events?

Reply:
Because `componentDidUpdate' will also trigger `updateSelection`. When I input a word using the ime, and then enter the second word quickly, `componentDidUpdate` will trigger during the process of entering the second word, and then move the cursor, causing the ime to exit.

Reply:
@yunwuxin but shouldn't when entering the second word it is back in an `isComposing` state?